types/Types.d.ts

```diff
@@ -0,0 +1,369 @@
+interface CommonPlaidLinkOptions {
+    onSuccess: Function;
+    onExit?: Function;
+    logLevel?: LinkLogLevel;
+    extraParams?: Record<string, any>;
+}
+
+export type LinkTokenConfiguration = (CommonPlaidLinkOptions & {
+    token: string;
+});
+
+export type LinkPublicKeyConfiguration = (CommonPlaidLinkOptions & {
+    publicKey: string;
+    token?: string;
+    clientName: string;
```

Comment:
Nullable on Android, non null on iOS. Not sure what to do here....

Reply:
It shouldn't be nullable on Android, public key config requires this